adapters/adview/adview.go

```diff
@@ -0,0 +1,167 @@
+package adview
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"text/template"
+
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/macros"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type adViewAdapter struct {
+	EndpointTemplate template.Template
+}
+
+// Builder builds a new instance of the Adf adapter for the given bidder with the given config.
```

Comment:
Looks like you've got a copy/paste typo here. Please replace "Adf" with "AdView".

Reply:
ok, fixed.